air/src/aux_table/bitwise_pow2/bitwise/mod.rs

```diff
@@ -279,6 +308,10 @@ trait EvaluationFrameExt<E: FieldElement> {
     fn bit_decomp(&self) -> &[E];
     /// Gets the entire range of decomposed input values for `a` and `b` in the next row.
     fn bit_decomp_next(&self) -> &[E];
+    /// Gets the value of the aggregated output in the previous row.
+    fn output_prev(&self) -> E;
+    /// Gets the value of the aggregated output in the next row of the previous row.
```

Comment:
I think "in the next row of the previous row" is a bit unclear. Can you rephrase for clarity?